modules/preview-react/side-panel/lib/NewSidePanel/SidePanelToggleButton.tsx

```diff
@@ -0,0 +1,70 @@
+/** @jsx jsx */
+import * as React from 'react';
+import {css, jsx} from '@emotion/core';
+import {space} from '@workday/canvas-kit-react/tokens';
+import {createComponent, useModelContext} from '@workday/canvas-kit-react/common';
+import {IconButton, IconButtonProps} from '@workday/canvas-kit-react/button';
+import {transformationImportIcon} from '@workday/canvas-system-icons-web';
+
+import {SidePanelModel, SidePanelModelContext, useSidePanelToggleButton} from './hooks';
+
+export interface SidePanelToggleButtonProps
+  extends Omit<IconButtonProps, 'aria-label'>,
+    React.ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
What's the reason for extending `React.ButtonHTMLAttributes` here? That should be handled automatically.

Reply:
Yeah, probably don't need to. I grabbed that from the current `SidePanel`.